Readme.md

```diff
@@ -329,17 +329,17 @@ makeDirs(['dir name 1', 'dir name 2']) // Create folders
 
 ### Install and uninstall npm or Yarn packages
 
-Installs npm package(s) and saves them to `package.json` if they aren’t installed yet or older than required.
+Installs npm package(s) and saves them to `package.json` if they aren’t installed yet or not satisfying range.
 
 ```js
 const { install } = require('mrm-core')
 install('eslint') // Install to devDependencies
 install(['tamia', 'lodash'], { dev: false }) // Install to dependencies
-install({ lodash: '4.17.3' }) // Install particular version
-install(['lodash'], { versions: { lodash: '4.17.3', other: '1.0.0' } }) // Install particular version
+install({ lodash: '^4.17.3' }) // Install particular version
+install(['lodash'], { versions: { lodash: '^4.17.3', other: '1.0.0' } }) // Install particular version
 ```
 
-**Note:** These versions are minimum required versions, Mrm will install the latest version if the current version is lower than required.
+**Note:** These should be valid semver ranges.
```

Comment:
Does it support all kinds of ranges like `>=1.2.3 <=2.3.4`?

Reply:
Yeas, I just did the test with a local runtime, that's working.
![capture d ecran 2018-08-08 a 16 25 46](https://user-images.githubusercontent.com/2330137/43843402-c72cbf5e-9b27-11e8-98da-ef32ba32e7b5.png)

Reply:
I'd add a few examples here: 

> Works with all semver ranges, like `1.2.3`, `^1.2.0` or `>=2`.